curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-a-typical-workday-and-tasks/657e5af613558ce1f6e1d501.md

```diff
@@ -9,7 +9,7 @@ dashedName: task-94
 
 A `security incident` is an event that may be harmful to the safety or security of a company or organization. For example, `A security incident like a stolen password needs to be reported immediately.`
 
-Note on `Harm` and `Harmful`:
+Note on `harm` and `harmful`: 
```

Comment:
why did you found that this space is necessary?

Reply:
I was trying to remove the failing tests.  I forget to remove it.